clubpoints.py

```diff
@@ -471,15 +471,20 @@ def class_point_parser(soup, event_date, mobile_format="standard"):
             continue
         car_number = int(item[2])
         position = first_element.replace("T", "")
-        if item[CT] in ["DNS", "DNF"]:
+        if item[CT] in ["DNS"]:
             continue
-        final_time = item[CT]
+        if item[CT] in ["DNF"]:
+            final_time = 0.000
+            points = 70
+        else:
+            final_time = float(item[CT])
         print(f"final time: {final_time}")
         if position == "1":
-            winner_time = float(item[CT])
+            winner_time = final_time
```

Comment:
Careful of detecting floats on strings - this indexed item may not always be a floating point